iceoryx_utils/include/iceoryx_utils/internal/relocatable_pointer/relative_ptr.hpp

```diff
@@ -253,34 +253,36 @@ class relative_ptr : public RelativePointer
         return *this;
     }
 
-    T& operator*()
+    template <typename SFINAE = T>
```

Comment:
it's weird to have the template parameter call `SFINAE`. It's more idiomatic to call it `U`

Reply:
You're probably right. Maybe add a comment regarding SFINAE and use `U`?

Reply:
I don't think a comment is necessary. The `std::enable_if` already states that SFINAE is used

Reply:
Sometimes I would like to have a more beautiful way to say "I want to have this function when a certain compile time condition is true". At the moment this is the best we can do, to my knowledge.

Reply:
Done.